### DISCUSSION TITLE
how do i verify the APK?

### DISCUSSION THREAD

**Discussion by stockh0lm** [Droid-ify/client#824] [Q&A]
I tried to install the APK using the AppVerifier, and this is what it shows as verification info:

com.looker.droidify
ED:88:59:C5:5A:F3:11:16:26:58:B9:4A:F9:82:B9:F0:91:DC:D2:76:28:D4:DE:34:86:D1:21:7E:BF:3C:99:35 

It expects something from the clipboard because there is a button labeled "Verify from clipboard." What info do I need to provide here for it to verify the app?

it would be helpful if you provided hashes or a gpg key or so.

**Comment by stockh0lm** [comment]
Apparently (based on info from a chat message) AppVerifier requires the signing certificate hash. so that would be nice if you could provide it.

**Reply by Iamlooker** [reply]
I will try adding it in this release
